app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,128 @@
  *= require_tree .
  *= require_self
  */
+
+.wrapper {
+  width: 60%;
+  margin-left: auto;
+  margin-right: auto;
+  padding-top: 15px;
+  border-top: 1px solid rgba(0,0,0,0.2)
+}
+
+.title {
+  font-family: "Times New Roman";
+  font-weight: 900;
+  text-align: center;
+  font-size: 40px;
+}
+
+.book_title {
+  font-family: "Times New Roman";
+  font-weight: 900;
+  font-size: 35px;
+}
+
+.book_author {
+  display: flex;
+  align-content: space-between;
+  font-size: 20px;
+  color: rgba(0,0,0,0.5);
+  display: inline-block;
+  padding-top: 5px;
+}
+
+.book_year {
+  font-size: 20px;
+  color: rgba(0,0,0,0.5);
+  padding-top: 10px;
+  padding-bottom: 10px;
+  border-bottom: 1px solid rgba(0,0,0,0.2)
+}
+
+.image {
+  float: left;
+  width: 250px;
+  height: 400px;
+  background-color: yellow;
+  margin-right: 50px;
+}
+
+a {
+  color: black;
+  font-weight: bold;
+  font-size: 12px;
+}
+
+a.request {
+  font-size: 20px;
+  font-family: Arial;
+  float: right;
+}
+
+table {
+  width: 55%;
+  margin-left: auto;
+  margin-right: auto;
+  border-collapse: collapse;
+  font-family: Arial;
+}
+
+tr {
+  line-height: 35px;
+  border-top: 1px solid rgba(0,0,0,0.2);
+}
+
+ul {
+    list-style-type: none;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    margin-top: auto;
+    margin-left: auto;
+    padding-bottom: 10px;
+    width: 90%;
+    border-bottom: 1px solid black;
+}
+
+li {
+  display: inline;
+}
+
+li a {
```

Comment:
Don't use html tags, use a class instead. You can use one level of nesting in sass http://sass-lang.com/guide